package.json

```diff
@@ -11,7 +11,7 @@
     "blockchain:contracts": "node --experimental-modules ./tests/e2e/setup-contracts.mjs",
     "build": "run-s check-types build:css build:app",
     "build:dev": "cross-env NODE_ENV=development npm run build",
-    "build:test": "cross-env NODE_ENV=test npm run build",
+    "build:test": "cross-env NODE_ENV=test NET=ropsten npm run build",
```

Comment:
we will need to re-issue our fixtures (if any) in goerli later?

Reply:
Yea, ropsten will be sunset soon by year end.

Reply:
mm ok lemme go through the testcafe fixtures and reissue them so we can avoid `ropsten` here. might need some time